specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualWan.json

```diff
@@ -2626,6 +2626,69 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/p2svpnGateways/{gatewayName}/getP2sVpnConnectionHealthDetailed": {
+      "post": {
+        "tags": [
+          "P2SVpnGateways"
+        ],
+        "operationId": "P2sVpnGateways_GetP2sVpnConnectionHealthDetailed",
+        "description": "Gets the sas url to get the connection health detail of P2S clients of the virtual wan P2SVpnGateway in the specified resource group.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "gatewayName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the P2SVpnGateway."
+          },
+          {
+            "name": "request",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/P2SVpnConnectionHealthRequest"
+            },
+            "description": "Parameters supplied to get the P2S Vpn connection health detail."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }          
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Follow the location header for sas-url to output blob."
```

Comment:
Should the location header be added for schema validation?

Reply:
Could you elaborate "the location header for schema validation"?

Reply:
I'll see if I can find the spec I say it in (but that also doesn't mean it's correct for ARM). It was something like:
```json
"200": {
   "descritption": "foo",
   "schema": {
      "header": {
        "headerName": "headerValue"
      }
   }
}
```
Rather than the regular `"schema": { "$ref": "#someType"}`

Reply:
Ah, here is one https://github.com/Azure/azure-rest-api-specs/blob/ab88105b1fd4d8b982869c43b1fbf365f24fc0af/specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json#L270-L284
Not sure since it's an older one, maybe that pattern isn't used anymore

Reply:
I searched it around.  All post APIs in network which requires pull on the location header, do not have such schema validation.

Reply:
@ninweninwe no worries
@number213 was that something you're tracking to change across the APIs?

Reply:
@nschonni I'll discuss it with the team, but since there aren't any response definitions in Network that describe it's headers, I think it will be low priority at best